ngraph_bridge/pass/transpose_sinking.cc

```diff
@@ -461,14 +469,18 @@ bool TransposeSinking::run_on_function(shared_ptr<ngraph::Function> f) {
   TransposeMap reorders, reuse_map;
   set<shared_ptr<ngraph::Node>> transposes_to_delete;
   unordered_map<std::string, ngraph::Shape> orig_result_out_shape;
+  if (std::getenv("NGRAPH_TF_DUMP_GRAPHS") != nullptr) {
+    NGRAPH_VLOG(0) << "Dumping ng_function before TS to Before_TS_"
+                   << f->get_name() << ".dot";
+    ngraph::plot_graph(f, "Before_TS_" + f->get_name() + ".dot");
+  }
```

Comment:
Should we dump these graphs in between the passes in the builder? It would be nice to have the function dumped after each pass à-la TF.

Reply:
We can do this in the builder.

Reply:
On second thought, I think it's better this way. In the builder we can do the "before_running_all_passes" but in between might not be possible since we run them all together.